lutris/game_actions.py

```diff
@@ -29,20 +29,26 @@
 
 
 def get_game_actions(game, window, application=None):
+    games = [game]
     if game.is_db_stored:
-        return GameActions(game, window, application)
+        return GameActions(games, window, application)
 
     if game.service:
-        return ServiceGameActions(game, window, application)
+        return ServiceGameActions(games, window, application)
 
-    return BaseGameActions(game, window, application)
+    return BaseGameActions(games, window, application)
+
+def get_multiple_game_actions(games, window):
```

Comment:
Suggest you combine this get_multiple_game_actions() and get_game_actions() into one method that takes a list. Then, if len(games)==1, it does the single-game code, else the multi-case.

Reply:
Tried that. But will need to modify the gamebar code too then. It doesn't pass a list and will cause problems.

Reply:
Yes indeed, but that should be a minimal change: get_game_actions([game]) instead of ...(game). Easy!

Reply:
Ok. Will try